CONTRIBUTING.md

```diff
@@ -1,14 +1,46 @@
-## Contribute to Tracim
+# Contribute
 
-In order to contribute to Tracim, please accept the licenses ([MIT](./LICENSE_MIT), [LGPLv3](LICENSE_LGPLv3) and [AGPLv3](LICENSE_AGPLv3)) and [Developer Certificate of Origin](./DCO).
+Thanks for taking the time to contribute :D
 
-To accept, sign all your commits with the `-s` arg of `git commit` command ([more info here](https://git-scm.com/docs/git-commit#git-commit--s)).
+## How to
 
-## Tests
+### Opening an issue
 
-- For backend tests, check [backend/README.txt]('./backend/README.txt)
-- For cypress tests, check [README.txt]('./README.txt')
+Before opening an issue, please check that the issue is not already opened.
+If there is no issue similar to yours, feel free to create an issue and follow the template.
 
+### Create a pull request
+
+Once you have finished your development and you want to merge it on `develop`, please create a pull request. The pull request must have a title explicit, for example: `feat(frontend): <your feature>`. Once the pull request is created, follow the template.
+
+### Contribute to the translation
+
+[Tracim](https://www.algoo.fr/fr/tracim) translation is done through [Weblate](https://weblate.org/tracim/tracim/).<br>
```

Comment:
`<br>` this is necessary here ?

Reply:
Everywhere there is a `<br>` I wanted a new line, since there is only two ways to create a new line without creating a new paragraph (`<br>` and `two space then enter`), I assumed this is a cleaner way. 

If you meant `necessary` in the way to keep the second line on the same line/paragraph, I just felt this way cleaner (this way = without big paragraph)

Reply:
Ok, good :)